code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
https://www.merriam-webster.com/dictionary/hem

Definition of hem
hemmed; hemming
intransitive verb
1
:  to utter the sound represented by hem hemmed and hawed before answering
2
:  equivocate the administration hemmed and hawed over the students' demands

4hemplay
interjection \usually read as ˈhem\
Definition of hem
—often used to indicate a vocalized pause in speaking